ipaclient/install/client.py

```diff
@@ -3287,6 +3285,11 @@ def _install(options, tdict):
             configure_sssd=options.sssd,
             force=options.force)
 
+        # Configure certmonger after krb5.conf is created and last
+        # to give higher chance that the new client is replicated.
+        configure_certmonger(fstore, subject_base, cli_realm, hostname,
+                             options, ca_enabled)
+
         logger.info("Configured /etc/krb5.conf for IPA realm %s", cli_realm)
```

Comment:
@rcritten, could you please move the logging up to right after configuring that conf?

Reply:
Good point, done.